src/models/network.js

```diff
@@ -128,6 +128,22 @@ Network.prototype.getNetworkStatus = function() {
 	return status;
 };
 
+Network.prototype.addChannel = function(newChan) {
```

Comment:
Could you add test(s) for this function in [`test/models/network.js`](https://github.com/thelounge/thelounge/blob/yamanickill/sort-channels-on-join/test/models/network.js)?

Reply:
Done